tsconfig.json

```diff
@@ -1,16 +1,42 @@
 {
   "compilerOptions": {
-    "outFile": "./dist/types/index",
+    "alwaysStrict": false,
+    "declaration": true,
+    "declarationMap": true,
+    "importHelpers": true,
+    "inlineSources": true,
+    "isolatedModules": true,
+    "lib": [
+        "ES2022",
+        "dom"
+    ],
+    "moduleResolution": "node10",
+    "sourceMap": true,
     "target": "ES6",
-    "lib": ["ES2022", "dom"],
-    "allowJs": true,
-    "forceConsistentCasingInFileNames": true,
-    "module": "NodeNext",
-    "moduleResolution": "NodeNext",
+    "skipLibCheck": true,
+    "esModuleInterop": true,
     "jsx": "react-jsx",
-    "declaration": true,
-    "emitDeclarationOnly": true
+    "emitDeclarationOnly": true,
+    "outDir": "./dist/types",
+    "ignoreDeprecations": "5.0"
   },
-  "include": ["**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules", "dist"]
-}
+  "files": [
+    "./src/index.ts",
+    "./src/types/Icon.types.ts",
+    "./src/Slider/Thumbs.tsx",
+    "./src/Slider/index.tsx",
+    "./src/SeparatorList/index.tsx"
```

Comment:
**Question**

Why not just import everything we need in `./src/index.ts`?

Reply:
yes this is a smarter approach and I've simplified how we include files in `tsconfig.json`